### DISCUSSION TITLE
cli: database migration:generate command

### DISCUSSION THREAD

**Discussion by noahjahn** [directus/directus#15197] [Feature Requests]
Add a new command to the Directus CLI that can generate migration files with the Directus standard identifier and name scheme.

For example, we could run the following command:

```
directus database migration:generate create-custom-table
```

Which will create a new file with scaffolded `up` and `down` exported functions in the `extensions/migrations` directory with an automatically generated identifier in the file name.

![directus-database-migration-generate](https://user-images.githubusercontent.com/36347998/185802920-9b58ef08-257c-4eac-8a64-3779ef0c72b3.gif)

The implementation is in a branch in my forked repo of directus: https://github.com/directus/directus/commit/93d757350b00061f12e9fbd02f5c4c0c96c3af86

**Comment by noahjahn** [comment]
I finally created a PR: https://github.com/directus/directus/pull/17738

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
